config_vm

```diff
@@ -0,0 +1,42 @@
+{
+	"boot-source": {
+		"kernel_image_path": "hello-vmlinux.bin",
+		"boot_args": "console=ttyS0 reboot=k panic=1 pci=off",
+		"initrd_path": null
+	},
+	"drives": [
+		{
+			"drive_id": "rootfs",
+			"path_on_host": "hello-rootfs.ext4",
+			"is_root_device": true,
+			"partuuid": null,
+			"is_read_only": false,
+			"cache_type": "Unsafe",
+			"io_engine": "Sync",
+			"rate_limiter": null
+		}
+	],
+	"machine-config": {
+		"vcpu_count": 2,
+		"mem_size_mib": 8192,
+		"smt": false,
+		"track_dirty_pages": false
+	},
+	"balloon": {
```

Comment:
It should work together as they operate on different memory regions (balloon on initial memory and virtio-mem on its own memory region).

But I changed it so it will be null for now (no belloon). No need to have them both while developing virtio-mem.